using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -555,16 +629,18 @@ Identity.getIdentifiers(new AdobeCallback<List<VisitorID>>() {
 {% endtab %}
 
 {% tab title="iOS" %}
-### getIdentifiers
+### getIdentifiers<a id="getIdentifiers-ios"></a>
 
-Returns all customer identifiers which were previously synced with the Adobe Experience Cloud.
+This `getIdentifiers` API returns all customer identifiers that were previously synced with the Adobe Experience Cloud
```

Comment:
- Update to "This API returns all..."
- Update solution name to "Experience Platform Mobile SDKs".